dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -80,3 +80,12 @@ preferCRLs.desc=By default, OCSP is the preferred mechanism for checking revocat
 
 disableFallback=No fallback
 disableFallback.desc=Disable the certificate revocation status checker fallback mechanism. During certificate revocation status check, the default behavior is fallback and check by using the non-preferred mechanism (OCSP or CRLs). Default is false.
+
+certCheckerSchedule=Certificate checker schedule
+certCheckerSchedule.desc=Time period for the ACME CA service to check if the certificate is expiring or revoked. If expiring or revoked, the certificate is renewed. If the first renew request does not work, the certificate renewal request continues until a new certificate is received using the timing set by certCheckerErrorSchedule.  Setting the certCheckerSchedule property to zero or a negative value disables automatic certificate renewal.
+
+certCheckerErrorSchedule=Certificate checker error schedule
+certCheckerErrorSchedule.desc=Time period for the ACME CA service to check if the certificate is expiring or revoked if the first request failed. If expiring or revoked, the certificate is renewed.
+
+disableMinRenewWindow=Disable minimum renew window
+disableMinRenewWindow.desc=Disables checking the minimum amount of time between certificate renew requests and allows immediate certificate renew requests.
```

Comment:
**change**
Disables checking the minimum amount of time between certificate renew requests and allows immediate certificate renew requests
**to**
Enables immediate certificate renewal requests by disabling the specified minimum amount of time between renewal requests

What sets the minimum that is disabled? Is a a default, or a value set by certCheckerErrorSchedule ?

Reply:
Updated